package.json

```diff
@@ -1,24 +1,25 @@
 {
   "name": "business-create-ui",
-  "version": "5.3.1",
+  "version": "5.4.0",
   "private": true,
   "appName": "Create UI",
   "sbcName": "SBC Common Components",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "lint": "eslint . --ext .js,.ts,.vue",
+    "test:unit": "vitest run",
+    "test:coverage": "vitest run --coverage"
   },
   "dependencies": {
     "@babel/compat-data": "^7.21.5",
     "@bcrs-shared-components/approval-type": "1.0.19",
+    "@bcrs-shared-components/base-address": "2.0.1",
     "@bcrs-shared-components/breadcrumb": "2.1.15",
-    "@bcrs-shared-components/business-lookup": "1.1.17",
+    "@bcrs-shared-components/business-lookup": "1.1.29",
```

Comment:
Updated this component as there's work there to fix an error corresponding to lodash.

Reply:
Would be nice to get rid of lodash eventually

Reply:
We could write the debounce function ourselves... in the future.

Please test that business-lookup works correctly, since it's been changed a bit since it was last used/tested here. Thx.

Reply:
BusinessLookup isn't working correctly. I don't think it has anything to do with Vite. Maybe recent changes broke it?

Reply:
I'm not getting any results back:
![no results](https://github.com/bcgov/business-create-ui/assets/122301442/e024d389-6cd1-4beb-be91-c70a462c4535)

Reply:
Are you getting any console errors?

Reply:
I wasn't. After updating the business lookup to version 1.1.30, I'm getting this error now:
![business lookup error](https://github.com/bcgov/business-create-ui/assets/122301442/f0189cb3-f4d2-42e5-89d7-5dc31e8dc93d)

Trying to find a fix...

Reply:
It's because the API key isn't populated.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/bcde34e9-e3ee-4edd-b06f-719d0990b6a5)

Reply:
Looks like there's some keys missing in ~/devops/cloudbuild-pr.yaml.

Reply:
Sev, thank you so so much. You're 100% right. By added them to cloudbuild-pr.yaml, the lookup was fixed. I'm assuming this'll also fix the AddressComplete.

Reply:
Phew! Glad to hear it.

I have no further comments, if someone wants to resolve this.

Reply:
`gcbrun` run doesn't work anymore 😁 
![gcbrun](https://github.com/bcgov/business-create-ui/assets/122301442/9c5175ab-4077-404d-9b2c-5f6387a9ff1c)

Reply:
The new keys have a typo.

Reply:
Still not working in the temporary URL. Let's leave it for now and I'll check if it's working when I deploy the branch to DEV. Is that fine?

Reply:
The API key in the API call request headers is still empty.

Yes, leave it for now.

Reply:
Is it still planned to temporary deploy this to Dev and test it there?

Reply:
Yes Sev. Currently, I see the Content Security Policy error. It's this and the address complete.